zilencer/management/commands/sync_api_key.py

```diff
@@ -16,9 +16,12 @@ def handle(self, *args, **options):
         config_file = os.path.join(os.environ["HOME"], ".zuliprc")
         if not os.path.exists(config_file):
             raise RuntimeError("No ~/.zuliprc found")
-        config = SafeConfigParser()
+        config = ConfigParser()
         with open(config_file, 'r') as f:
-            config.readfp(f, config_file)
+            if hasattr(config, 'read_file'):
+                config.read_file(f, config_file)
+            else:
+                config.readfp(f, config_file)
```

Comment:
@timabbott Okay

This is when I try in python2:
```
Python 2.7.12 (default, Sep 29 2016, 13:30:34) 
[GCC 6.2.1 20160916 (Red Hat 6.2.1-2)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from six.moves.configparser import ConfigParser
>>> 
>>> config = ConfigParser()
>>> config.readfp
<bound method ConfigParser.readfp of <ConfigParser.ConfigParser instance at 0x7f00680559e0>>
>>> config.read_file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: ConfigParser instance has no attribute 'read_file'
```

Meanwhile in python3:
```
Python 3.5.2 (default, Oct 14 2016, 12:54:53) 
[GCC 6.2.1 20160916 (Red Hat 6.2.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from six.moves.configparser import ConfigParser
>>> 
>>> config = ConfigParser()
>>> config.readfp
<bound method RawConfigParser.readfp of <configparser.ConfigParser object at 0x7f3bce5e47b8>>
>>> config.read_file
<bound method RawConfigParser.read_file of <configparser.ConfigParser object at 0x7f3bce5e47b8>>
>>> 
```

So the conclusion here is, `six.moves.configparser.ConfigParser` doesn't have `read_file` attribute when we try in python2. But in python3 it has that attribute..